includes/embeds/class-amp-youtube-embed-handler.php

```diff
@@ -151,10 +245,59 @@ public function render( $args, $url ) {
 				'width'       => $this->args['width'],
 				'height'      => $this->args['height'],
 				'placeholder' => '',
+				'start'       => 0,
 			]
 		);
 
-		if ( empty( $args['video_id'] ) ) {
+		$attributes = array_merge(
+			[ 'data-videoid' => $args['video_id'] ],
+			wp_array_slice_assoc( $args, [ 'layout', 'width', 'height' ] )
+		);
+		if ( ! empty( $args['title'] ) ) {
+			$attributes['title'] = $args['title'];
+		}
+
+		$query_vars  = [];
+		$query_param = wp_parse_url( $url, PHP_URL_QUERY );
+		wp_parse_str( $query_param, $query_vars );
+		$query_vars = ( ! empty( $query_vars ) && is_array( $query_vars ) ) ? $query_vars : [];
```

Comment:
An `is_array()` check is sufficient, no?

```suggestion
		$query_vars = is_array( $query_vars ) ? $query_vars : [];
```